config/bravo_sdf.cfg

```diff
@@ -39,3 +39,15 @@ lcio_bin_jar = /sdf/home/b/bravo/.m2/repository/org/lcsim/lcio/2.7.4-SNAPSHOT/lc
 [EvioToLcio]
 hps_java_bin_jar = /sdf/home/b/bravo/.m2/repository/org/hps/hps-distribution/4.5-SNAPSHOT/hps-distribution-4.5-SNAPSHOT-bin.jar
 conditions_url = jdbc:sqlite:/sdf/group/hps/users/bravo/db/hps_conditions_test.db
+
+[PEDE]
+#command = /full/path/not/in/PATH/pede
```

Comment:
Require `pede` to be in PATH - update install method to make sure pede's path is in PATH